.circleci/config.yml

```diff
@@ -2,10 +2,82 @@ version: 2.1
 orbs:
   architect: giantswarm/architect@0.2.0
 
+e2eTest: &e2eTest
+  machine: true
+  working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/metrics-server-app
+  steps:
+    - checkout
+
+    - attach_workspace:
+        at: .
+
+    - run:
+        name: Get e2ectl binary
+        command: |
+          wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2ectl/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+          chmod +x ./e2ectl
+    - run:
+        name: Init cluster
+        command: |
+          ./e2ectl cluster create
+          cp $(./e2ectl kubeconfig path) ${E2E_TEST_DIR}/kubeconfig
+
+    - run:
+        name: Run test
+        command: |
+          docker run --rm --network host \
+            -v $(pwd)/${E2E_TEST_DIR}:/e2e \
+            -v $(pwd):/go/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} \
+            -e E2E_TARBALL_URL=https://giantswarm.github.io/default-test-catalog/metrics-server-app-`./architect project version`.tgz\
```

Comment:
We tend to used `$(./architect project version)` instead of:
```
`./architect project version]`
```

Reply:
Thanks, I would replace it.